src/components/common/UploadDocumentEwallet.tsx

```
@@ -1,108 +1,442 @@
 import { AuthContext } from '../../utils/context/checkToken';
 import CommonButton from './button/Button';
 import { useTranslation } from 'react-i18next';
+import { Box, Text, Alert, AlertIcon, IconButton, useToast, Progress } from '@chakra-ui/react';
+import { ArrowBackIcon } from '@chakra-ui/icons';
+import { mockUploadDocumentPayload, mockUploadDocumentPayload2 } from '../../assets/mockdata/DocumentMockData';
 
 const VITE_EWALLET_ORIGIN = import.meta.env.VITE_EWALLET_ORIGIN;
 const VITE_EWALLET_IFRAME_SRC = import.meta.env.VITE_EWALLET_IFRAME_SRC;
-const UploadDocumentEwallet = ({ userId }) => {
+
+const UploadDocumentEwallet = ({ userId }: { userId: string }) => {
```

ESLint

[error] 'userId' is defined but never used.

(@typescript-eslint/no-unused-vars)